rootconf/default/bin/exec_installer

```diff
@@ -252,44 +252,43 @@ url_run()
     return 1
 }
 
-# Try various HTTP URLs
-http_download()
+get_onie_neighs()
 {
-    # Build list of HTTP servers to try
-    http_servers="$onie_server_name"
-
-    # HTTP server IP only
-    if [ -n "$onie_disco_wwwsrv" ] ; then
-        http_servers="$http_servers $onie_disco_wwwsrv"
-    fi
-
-    # Try BOOTP next-server IP as HTTP
-    if [ -n "$onie_disco_siaddr" ] ; then
-        http_servers="$http_servers $onie_disco_siaddr"
-    fi
-
-    # Try DHCP server IP as HTTP
-    if [ -n "$onie_disco_serverid" ] ; then
-        http_servers="$http_servers $onie_disco_serverid"
-    fi
-
-    # Try TFTP server IP as HTTP
-    if [ -n "$onie_disco_tftpsiaddr" ] ; then
-        http_servers="$http_servers $onie_disco_tftpsiaddr"
-    fi
-
-    # Add link local neighbors
     while [ ${#onie_neighs} -gt 0 ] ; do
         n=${onie_neighs%%,*}
         onie_neighs=${onie_neighs#*,}
         tmp=$(echo $n | sed -e 's/-/%/g');
-        http_servers="$http_servers $tmp"
+        echo "$tmp"
```

Comment:
instead of storing the result in `tmp` and echoing `tmp`, just return the output of the sed command:
```
echo $n | sed -e 's/-/%/g'
```